controllers/imagecollector/imagecollector_controller.go

```diff
@@ -323,7 +323,15 @@ func (r *Reconciler) createImageJob(ctx context.Context) (ctrl.Result, error) {
 		pullSecrets = append(pullSecrets, corev1.LocalObjectReference{Name: secret})
 	}
 
+	labels := map[string]string{}
+	for k, v := range mgrCfg.AdditionalPodLabels {
+		labels[k] = v
+	}
+
 	jobTemplate := corev1.PodTemplateSpec{
+		ObjectMeta: metav1.ObjectMeta{
+			Labels: labels,
+		},
```

Comment:
since we are also setting this in the imagejob_controller we can get rid of the functionality here

Reply:
I have removed it.